Scripts/Skills/DetectHidden.cs

```diff
@@ -191,10 +191,6 @@ public static bool CanDetect(Mobile src, Mobile target)
             if (target.Blessed || (target is BaseCreature && ((BaseCreature)target).IsInvulnerable))
                 return false;
 
-            // pet owner, guild/alliance, party
-            if (!Spells.SpellHelper.ValidIndirectTarget(target, src))
```

Comment:
THis cannot be removed. Blue players in Trammel cannot detect other blue players. THis would cause griefing.